src/components/navigation/FlashNotification.tsx

```diff
@@ -1,48 +1,44 @@
 import * as React from 'react'
-import { Text, View } from 'react-native'
+import { View } from 'react-native'
 import { AirshipBridge } from 'react-native-airship'
 import AntDesignIcon from 'react-native-vector-icons/AntDesign'
 
-import { THEME } from '../../theme/variables/airbitz'
 import { AirshipDropdown } from '../common/AirshipDropdown'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText, Paragraph } from '../themed/EdgeText'
 
 interface Props {
   bridge: AirshipBridge<void>
   message: string
+  icon?: React.ReactNode
   onPress?: () => void
 }
 
 export function FlashNotification(props: Props) {
-  const { bridge, message, onPress = () => {} } = props
+  const { bridge, message, icon, onPress = () => {} } = props
   const theme = useTheme()
   const styles = getStyles(theme)
 
   return (
     <AirshipDropdown bridge={bridge} backgroundColor={theme.modal} onPress={onPress}>
       <View style={styles.container}>
-        <AntDesignIcon name="checkcircle" size={theme.rem(2)} style={styles.icon} />
-        <Text style={styles.text}>{message}</Text>
+        {icon ?? <AntDesignIcon name="checkcircle" size={theme.rem(2)} style={styles.icon} />}
+        <Paragraph center>
+          <EdgeText>{message}</EdgeText>
```

Comment:
Do we need `EdgeText` inside `Paragraph`? I thought  the`<Paragraph>` was enough by itself